docs/rag_guide.md

```
@@ -357,8 +357,18 @@ okp:
   chunk_filter_query: "product:*openshift*"
 ```
 
-> [!NOTE]
-> This static filter is a temporary work-around until dynamic per-request filtering is supported.
+Per-request filtering is also available on all inference endpoints via request field **`solr`**: `mode` (`semantic`, `hybrid`, or `lexical`) and `filters` (key:value format). Legacy payloads that omit `mode`/`filters` and send filter key:value pairs at the top level still work with `mode` set to `hybrid`. 
+
+Example:
+```json
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)